packages/devui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "devui",
-  "version": "1.0.0-8",
+  "version": "1.0.0-9",
```

Comment:
Can you revert the version bump? I'll bump it when I got to deploy it.

Reply:
ok